core/app/[locale]/(default)/(auth)/change-password/loading.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Loading() {
+  return null;
+}
```

Comment:
Not great. I'd prefer if the server simply blocked while the dynamic page is rendered instead of immediately returning an empty loading state followed by streaming in the dynamic page later.